### PR TITLE
add apiary_mock_server blog post

### PR DESCRIPTION
**Pull request by jackblandin**
review @RossAllenBell @jeanbahnik

### CODE REVIEW

File: _posts/2015-10-09-develop-faster-with-apiary.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+title: ! 'Develop faster with Apiary'
+author: Jack Blandin
+categories:
+- javascript
+---
+
+When building a front-end application in a microservice architecture you may find yourself waiting for other developers to complete the back-end so that you know what endpoints you have to work with. Here I will discuss how to use [Apiary](https://apiary.io/) to reduce this waiting time.
+
+### Apiary
+Apiary is a tool used for designing, documenting, and testing API's. In addition, it has a tool called a _mock server_ which is described on their [website](https://apiary.io/) as
```
Comment (jeanbahnik) [inline _posts/2015-10-09-develop-faster-with-apiary.md]: APIs, not API's

**Comment by jeanbahnik** [comment]
Looks good! Ends a bit abruptly. Maybe add how you've used it, how you plan on using it, or maybe what feature you'd like to see added to it.

**Comment by RossAllenBell** [comment]
LGTM

File: _posts/2015-10-09-develop-faster-with-apiary.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: ! 'Develop faster with Apiary'
+author: Jack Blandin
+categories:
+- javascript
+---
+
+When building a front-end application in a microservice architecture you may find yourself waiting for other developers to complete the back-end so that you know what endpoints you have to work with. Here I will discuss how to use [Apiary](https://apiary.io/) to reduce this waiting time.
```
Comment (RossAllenBell) [inline _posts/2015-10-09-develop-faster-with-apiary.md]: Actually, it sounds like you're writing a school assignment when you say

> Here I will discuss how to use [Apiary](https://apiary.io/) to reduce this waiting time.

I'd go with something a little more conversational.

**Pull request merged**